source/_components/sensor.sensehat.markdown

```diff
@@ -34,5 +35,119 @@ Configuration variables:
   - 'temperature'
   - 'humidity'
   - 'pressure'
+  is_hat_attached (Optional): True|False boolean; True declares the SenseHAT _is_ physically on the Raspberry Pi
+  
 
+#### Customizing the Sense HAT data
```

Comment:
I should add, that that happened with the merger, not the way I wrote it. To correct it all, this version should be used. (_or let me fix / merge_)